controllers/managedocs_controller_test.go

```diff
@@ -640,13 +646,18 @@ var _ = Describe("ManagedOCS controller", func() {
 			})
 			When("there is a valid size in the add-on parameter secret", func() {
 				It("should create reconciled resources", func() {
-					if testReconciler.DeploymentType == consumerDeploymentType {
-						Skip(fmt.Sprintf("Skipping the test as it is not required by %v", testReconciler.DeploymentType))
-					}
+
```

Comment:
- shouldn't this test be skipped for converged?

Reply:
No, this test should be for all the deployment types. The resources that gets created might be different depending on deployment types. And the storageCluster spec is different depending on deployment type.